bundles/org.openhab.binding.sonos/README.md

```diff
@@ -69,7 +69,7 @@ The devices support the following channels:
 | currenttrack        | String    | R           | Name of the current track or radio station currently playing                                                                                              | all                                  |
 | currenttrackuri     | String    | R           | URI of the current track                                                                                                                                  | all                                  |
 | currenttransporturi | String    | R           | URI of the current AV transport                                                                                                                           | all                                  |
-| favorite            | String    | W           | Play the given favorite entry. The favorite entry has to be predefined in the Sonos Controller app                                                        | all                                  |
+| favorite            | String    | W           | Play the given favorite entry. The favorite entry has to be predefined in the Sonos Controller app. Use the exact name of the favorite shown in the app for start playing from rules.                                                      | all                                  |
```

Comment:
Probably rather "... shown in the app **to** start playing..."
By "app", you mean the Sonos app ?

Reply:
yes, thanks for the feedback. this is my first addition to the documentation. I'm learning every day :)